content/docs/start/data-and-model-versioning.md

```diff
@@ -86,6 +85,12 @@ outs:
 
 </details>
 
+> Note that there are [other mechanisms] to track data files/dirs besides `.dvc`
```

Comment:
my 2cs - it expands the start page (we should be optimizing for its size) and I'm not sure it solves the problem. It's not informative enough. 

The expectation is that at this stage people are not familiar with pipelines yet. This reference would confuse at best (how important it is to know? should I jump and start reading the second doc?)

I would clarify how did that happen that user was familiar with pipelines but didn't know about them tracking files automatically? May be we should change the pipelines docs instead.

If we want to keep this note, I would make more friendly, less obscure:

Note, when you start using pipelines `dvc add` is needed only for pipeline inputs, usually data. Everything else is handled ...

Reply:
While this only added one line so I think the size impact isn't that important, now that I read it a week later I do agree that it doesn't necessarily clarify anything. I'll revert and check whether the pipelines page needs the clarification instead... ⌛